selfdrive/car/subaru/interface.py

```diff
@@ -107,6 +107,21 @@ def _get_params(ret, candidate, fingerprint, car_fw, experimental_long, docs):
     else:
       raise ValueError(f"unknown car: {candidate}")
 
+    #ret.flags |= SubaruFlags.EXPERIMENTAL_LONG.value
+    experimental_long_allowed = (ret.flags & SubaruFlags.EXPERIMENTAL_LONG) != 0
+
+    # longitudinal
+    ret.experimentalLongitudinalAvailable = experimental_long_allowed and candidate not in (GLOBAL_GEN2 | PREGLOBAL_CARS | LKAS_ANGLE)
```

Comment:
make sure to comment this out before merging

Reply:
done